src/main/java/com/java_essential/Cherednichenko/extra_task/CustomerService/SupportService.java

```diff
@@ -0,0 +1,103 @@
+package com.java_essential.Cherednichenko.extra_task.CustomerService;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class SupportService {
+    Customer[] customers = new Customer[10];
+    Profile[] profiles = new Profile[13];
+    Service[] services = new Service[5];
+    SupportSpecialist[] supportSpecialists = new SupportSpecialist[3];
+    Tiket[] tikets = new Tiket[10];
+
+    public static void main(String[] args) {
+
+    }
+
+    public void fetch_all_customers() {
+        for (int i = 0; i <= customers.length - 1; i++) {
+            System.out.println("Customer " + customers[i].getUsername());
```

Comment:
Method call fetch_all_customers and other methods can't demonstrate relationships among classes, moreover firstly you should create this relationships. See slack conversation about it in channel 'homework'